src/app/Http/Controllers/CrudFeatures/AjaxTable.php

```diff
@@ -24,7 +25,7 @@ public function search()
                     ->toArray();
 
         // structure the response in a DataTable-friendly way
-        $dataTable = new \LiveControl\EloquentDataTable\DataTable($this->crud->query, $columns);
+        $dataTable = new DataTable($this->crud->query, $columns);
```

Comment:
TBH I'm not sure why not specifying the full  name is cleaner, although I suppose it might help if someone wants to replace `DataTable` with something else for some (probably obscure) reason.